### ISSUE TITLE
Possible improvements for deck list

### ISSUE THREAD

**Issue by akaan** [zzorba/ArkhamCards#135]
I was thinking of some improvements for the deck list. @zzorba I know you're busy implementing other things but I thought I might just list them here for later.

- when browsing through the deck list, some kind of visual indicator to differenciate ArkhamDB decks from "offline" decks (an 🌐 icon next to the deck name or near the update date)
- There's already a "Hide ArkhamDb Decks" option, maybe add the inverse option "Hide non-ArkhamDB decks"
- Similarly, an option to hide decks included in a campaign
- With our collection of decks growing, a way to organize them would be nice : I don't what will hand out the best user experience : folders, keywords / tags

As always, many thanks for your hard work of this wonderful app !

**Comment by zzorba** [comment]
Yeah, this is good feedback.  Currently working through the last bits of campaign uploading/sharing, so it might be a while before I can address this stuff.  But I'll revisit for sure once this is done.

I've long toyed with the idea of 'archiving' items as a sort of... first take on folders/labels.  At least for campaigns you finish that is.
